e2e/e2etests/test_erc20_multiple_withdraws.go

```diff
@@ -10,15 +10,36 @@ import (
 	testcontract "github.com/zeta-chain/zetacore/testutil/contracts"
 )
 
-func TestMultipleWithdraws(r *runner.E2ERunner) {
+func TestMultipleWithdraws(r *runner.E2ERunner, args []string) {
+	approvedAmount := big.NewInt(1e18)
+	if len(args) != 2 {
+		panic("TestMultipleWithdraws requires exactly two arguments: the withdrawal amount and the number of withdrawals.")
+	}
+
+	withdrawalAmount, ok := big.NewInt(0).SetString(args[0], 10)
+	if !ok || withdrawalAmount.Cmp(approvedAmount) >= 0 {
+		panic("Invalid withdrawal amount specified for TestMultipleWithdraws.")
+	}
+
+	numberOfWithdrawals, ok := big.NewInt(0).SetString(args[1], 10)
+	if !ok || numberOfWithdrawals.Int64() < 1 {
```

Comment:
We could check for a max number of withdrawals to avoid an excessive number.

Reply:
Yes, I saw the amount limit. Not a big problem but there could be many small withdrawals pointed at some production network. I will approve anyway but we can address this later if it is an issue.